### PR TITLE
Update tracking language

### PR DESCRIPTION
**Pull request by silent1mezzo**
Also fixes formatting

### CODE REVIEW

File: pages/faq.tsx [hunk 1]
```diff
@@ -29,132 +32,212 @@ export default function FAQ() {
...
                   <AccordionIcon />
                 </AccordionButton>
               </h2>
               <AccordionPanel pb={4}>
-                You have full control over who your information will be shared with — nobody else will have access to any personally identifiable information. We take your privacy seriously and have taken this into consideration at every step. The site does not use any third-party tracking or analytics tools.
+                You have full control over who your information will be shared with — nobody else
+                will have access to any personally identifiable information. We take your privacy
+                seriously and have taken this into consideration at every step. This site does not
+                use cookies or any third-party tracking tools. We use{' '}
+                <a
```
Comment (JamiesonRoberts) [inline pages/faq.tsx:117]: If you use `<Link href='...' isExternal />` the isExternal prop will take care of the target and rel for you!

File: pages/faq.tsx [hunk 2]
```diff
@@ -29,132 +32,212 @@ export default function FAQ() {
...
               </AccordionPanel>
             </AccordionItem>
 
             <AccordionItem>
               <h2>
                 <AccordionButton>
                   <Box flex="1" textAlign="left">
-                    <Heading as="h2" size="sm">Why might the media want to follow-up with storytellers? Is MyCovidStory.ca a direct line to media coverage?</Heading>
+                    <Heading as="h2" size="sm">
+                      Why might the media want to follow-up with storytellers? Is MyCovidStory.ca a
```
Comment (JamiesonRoberts) [inline pages/faq.tsx:134]: I think we have most of the site normalized to have COVID in all caps wherever it appears (except for like a text representation of the URL)

**Review by JamiesonRoberts — COMMENTED** [review]
Looks good, couple of comments/ideas

File: pages/faq.tsx (hunk 2 above)
Comment (silent1mezzo) [reply pages/faq.tsx:134]: @JamiesonRoberts based off other places this is standardized? I think it's ok because it's our URL but otherwise agree.

Comment (JamiesonRoberts) [reply pages/faq.tsx:134]: Yeah thats fair!

File: pages/faq.tsx (hunk 1 above)
Comment (silent1mezzo) [reply pages/faq.tsx:117]: As far as I can tell <Link> is meant for internal links?

Comment (JamiesonRoberts) [reply pages/faq.tsx:117]: Then why would they have a prop setting the external target and rel? Either way, not a huge deal :)

**Review by JamiesonRoberts — APPROVED** [review]

**Pull request merged**